homeassistant/components/vizio/config_flow.py

```diff
@@ -33,11 +40,20 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def _get_config_schema(input_dict: Dict[str, Any] = None) -> vol.Schema:
+def _get_config_schema(
+    input_dict: Dict[str, Any] = None, apps: Dict[str, Any] = None
+) -> vol.Schema:
     """Return schema defaults for config data based on user input/config dict. Retain info already provided for future form views by setting them as defaults in schema."""
     if input_dict is None:
         input_dict = {}
 
+    if apps is None:
+        apps = {}
+
+    default_include_or_exclude = (
+        CONF_EXCLUDE if CONF_EXCLUDE in apps.keys() else CONF_INCLUDE
```

Comment:
.keys() is the default.

```suggestion
        CONF_EXCLUDE if CONF_EXCLUDE in apps else CONF_INCLUDE
```